packages/theme/pages/MyAccount.vue

```diff
@@ -142,6 +142,11 @@ export default {
   }
   &__logout {
     cursor: pointer;
+    outline: none;
+    border: none;
+    background: transparent;
+    padding: 0;
+    font-size: inherit;
```

Comment:
I believe we should create a reusable TextButton component that extends the [SfButton](https://docs.storefrontui.io/?path=/docs/components-atoms-button--primary) by implementing custom styles instead of defining it locally in the component scope.

Reply:
Extracting it to a separate component seems right but in my opinion that component would be so simple that it doesn't have to extend SfButton (a basic styled button with 'click' emit seems enough). What do you think?

Reply:
A draft PR of that: https://github.com/vuestorefront/spree/pull/280